creator-node/src/services/stateMachineManager/stateMachineUtils.js

```diff
@@ -65,10 +66,10 @@ const retrieveClockStatusesForUsersAcrossReplicaSet = async (
 
         // Sign request to other CN to bypass rate limiting
         const { timestamp, signature } = generateTimestampAndSignature(
-          { spID: SP_ID },
+          { spID: spID },
```

Comment:
nit: these could all just be `spID` instead of `spID: spID` but fine if you want to leave it as is

Reply:
ah ye just saw this comment, whoops - will fix as followup